dotcom-rendering/src/model/decideCollectionBranding.ts

```diff
@@ -0,0 +1,42 @@
+import { isNonNullable } from '@guardian/libs';
+import type { EditionId } from '../lib/edition';
+import type { Branding } from '../types/branding';
+import type { FEFrontCard } from '../types/front';
+
+const getBrandingFromCards = (
```

Comment:
Duplicate from [here](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/model/enhanceCollections.ts#L32). The aim is to delete the method from `enhanceCollections.ts` but at this point I could not import it because I was getting a "circular dependency" error as `enhanceCollections.ts` imports `decideCollectionBranding`